api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -421,7 +421,7 @@ type HostedClusterSpec struct {
 	// SingleReplica.
 	//
 	// +optional
-	// +kubebuilder:default:="SingleReplica"
+	// +kubebuilder:default:="HighlyAvailable"
```

Comment:
You simply can't do that.
This API is released as v1beta1, every change has to be backward compatible and this is not the case here.
That's why I raised my concerns.